packages/plugins/GeoLocation/src/store/getters.ts

```diff
@@ -28,6 +28,9 @@ const getters: PolarGetterTree<GeoLocationState, GeoLocationGetters> = {
     }
     return false
   },
+  renderType: (_, __, ___, rootGetters): string => {
+    return rootGetters.configuration?.geoLocation?.renderType || 'independent'
+  },
```

Comment:
Return Type should be `RenderType` from the types-package

Reply:
Yep. https://github.com/Dataport/polar/pull/21/commits/80762b2257660d8b43a71cc1d74b032fea970fcb